Lib/test/test_concurrent_futures/test_wait.py

```diff
@@ -126,6 +126,9 @@ def test_timeout(self):
                 timeout=short_timeout,
                 return_when=futures.ALL_COMPLETED)
 
+        event.set()
+        futures.wait([future], return_when=futures.ALL_COMPLETED)
```

Comment:
Those two lines are just to be sure that we correctly cleanup `future` or is it for something else? (since it was only pending in the previous `wait()`)

Reply:
I misread the code below. Yeah, we just need to set the event so that job can finish. I'll rearrange it slightly.